walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java

```diff
@@ -50,7 +50,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Task task = new Task(id, title, timestamp);
       tasks.add(task);
     }
-
```

Comment:
It looks like the only change in this file is removing this empty line. please undo since it doesn't add anything

you can use git diff to look at the changes you're committing.